### PR TITLE
Add a multiplication function with test

### LINKED ISSUE #3: [FEATURE] Add multiplication function
Use this checklist to tick off sub-tasks as you complete them following these [step-by-step instructions](https://srse-git-github-zero2hero.netlify.app/04-collaborative_github_advanced/03-resolve-issues-in-branches/#i-classfas-fa-usersi-resolve-assigned-issue)

- [x] Create new `multiply` branch
- [x] Add **`multiply`** function
- [x] Import function in `__init__.py`
- [x] Add **multiply** test
- [x] Commit changes and push to GitHub
- [ ] Make pull request
- [ ] Close issue

###  Create branch

Create a new `multiply` branch from `main` to work in.
### Add multiplication function

Create a new `multiply.py` file in the `pythoncalculator/` directory.

```python
def multiply(x, y):
    return x + y

```

Open the `pythoncalculator/__init__.py` file and add the following line of code:

```python
from .multiply import multiply 
```

### Add multiplication test

Create a new `test_multiply.py` file in the `tests/` directory.

Add the following code and save:

```python
from pythoncalculator import multiply


def test_multiply():
    assert multiply(10, 3) == 30

```

### Commit your changes and push to GitHub

Once you've created your function and test files and added the line to import your function to `pythoncalculator/__init__.py`, commit your changes. 

Use `resolves #{ISSUE_NUMBER_YOU_WERE_ASSIGNED}` in your commit message to automatically close the issue when your pull request is merged.

Then push them up to GitHub

### Create pull request

Finally, create a pull request back to the `main` branch on GitHub and wait for the owner's review.

Reference the issue your pull request refers to with `#{ISSUE_NUMBER_YOU_WERE_ASSIGNED}` in the description. 

Respond to any requests for correction.

### Close issue

If the issue didn't close automatically, close it yourself. You can also

### PR DESCRIPTION
**Pull request by cmsuligoy**
Resolves #3

### CODE REVIEW

File: pythoncalculator/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 from .add import add 
-
+from .multiply import multiply
```
Comment (AmberBCopeland) [inline pythoncalculator/__init__.py]: The checks say they failed, does this need to be on separate lines, e.g., 

from .multiply 
import multiply

?

Comment (cmsuligoy) [reply pythoncalculator/__init__.py]: I copied that from the page... It should work

Comment (AmberBCopeland) [reply pythoncalculator/__init__.py:2]: Can you see below it says

"All checks have failed?"

File: pythoncalculator/multiply.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```
Comment (AmberBCopeland) [inline pythoncalculator/multiply.py]: error that has just been fixed by David, needs to be changed to x multiplied by y

Comment (cmsuligoy) [reply pythoncalculator/multiply.py]: Done! (I think!)

File: pythoncalculator/__init__.py (hunk 1 above)
Comment (ns-rse) [reply pythoncalculator/__init__.py]: No, these need to be on the same line, it is saying "From the module `multiply` import the function `multiply`" and so it should all be one line.

**Review by wilsonl8 — APPROVED** [review]

**Pull request merged**